### PR TITLE
First draft of the three carrot home office assessment

### PR DESCRIPTION
**Pull request by lewisnyman**
If anyone has any more ideas of suggestions we could add to this then please do :smile: 

It would be great to get some feedback from @huntons on the comments on the broadband tax sentence.

### CODE REVIEW

File: working-at-wunderkraut/health-and-safety/work-comfortably/home-office-assessment.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+If you work from home, you have both power and responsibility over your personal work environment. You can tweak it and customize it to suit your needs. But it can be easy to overlook the essentials.
+
+This three carrot rating system makes it simple to assess the quality of your work environment.
+
+## One carrot: The essentials
+
+* An internet connection [at least 1Mbps up and 2Mbps down.](https://support.google.com/plus/answer/1216376?hl=en) [Take a speed test](http://www.speedtest.net/)
+  * Fix this by upgrading your broadband. this is not expensable for tax reasons, but is already factored into your salary. Think of it as the cost of getting to work, but instead commuting by train or bus you take the internet, which is a lot cheaper and quicker for you. In just the same way you need your method of getting to work to be fairly reliable.
```
Comment (joesb) [inline working-at-wunderkraut/health-and-safety/work-comfortably/home-office-assessment.md]: Sorry, but there's a little typo:

**this** is not expensable for tax reasons » **This** is not …

File: working-at-wunderkraut/health-and-safety/work-comfortably/home-office-assessment.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## Two carrots: Productivity
+
+* A quiet and distraction free work environment.
+* Good natural lighting
+* Your optimal working temperature
+
+## Three carrots: Optimal comfort
+
+* An internet connection at least 2.6Mbps up and 4.0Mbps down.
+* A heathier chair and/or desk option. For example: Standing desk, exercise ball, or [see our recommended equipment.](/working-at-wunderkraut/health-and-safety/work-comfortably/recommended-equipment/)
```
Comment (joesb) [inline working-at-wunderkraut/health-and-safety/work-comfortably/home-office-assessment.md]: Sorry, but there's a little typo here also:

A **heathier** chair » A **healthier** chair

**Comment by joesb** [comment]
Hmm, looks like this branch/PR has conflicts now @lewisnyman - could you review the last comments and the conflict? Let me know when you're happy and I can review the PR again.

**Comment by lewisnyman** [comment]
This should be mergable now

**Comment by joesb** [comment]
Ok, I guess typos can be dealt with some other time. Let's merge this.

**Pull request merged**